packages/suite/src/components/suite/modals/Passphrase/components/PassphraseTypeCard/index.tsx

```diff
@@ -1,131 +1,189 @@
-import React, { useState, createRef, useLayoutEffect } from 'react';
+import React, { useState, createRef } from 'react';
+import { AnimatePresence, motion } from 'framer-motion';
+import { ANIMATION } from '@suite-config';
 import { useKeyPress } from '@suite-utils/dom';
 import styled, { css } from 'styled-components';
-import { Button, colors, variables, Input, Checkbox, Icon } from '@trezor/components';
+import { Button, colors, variables, Input, Tooltip, Checkbox, Icon } from '@trezor/components';
 import { Translation } from '@suite-components/Translation';
-
 import { MAX_LENGTH } from '@suite-constants/inputs';
 import { countBytesInString } from '@suite-utils/string';
 import PasswordStrengthIndicator from '@suite-components/PasswordStrengthIndicator';
 
-const WalletTitle = styled.div`
-    font-size: ${variables.FONT_SIZE.NORMAL};
-    text-align: center;
-    color: ${colors.BLACK0};
-    margin-bottom: 12px;
-`;
-
-const Col = styled.div<Pick<Props, 'colorVariant' | 'singleColModal'>>`
+const Wrapper = styled.div<Pick<Props, 'type' | 'singleColModal'>>`
```

Comment:
This is Props type on line 162? I was thinking for a while where it comes from and why linter does not bitch about "used before it was defined" like it does with functions.

Reply:
Yup, it doesn't do that. We can even move all styled-components below the component function as it would be more convenient to see the render function before all css stuff. (We probably will do that once someone finds enough willpower to refactor it across the codebase)

Reply:
Personally I have adopted convention to have all styled components  appear before the component itself :D I don't really care.